roles/kube_vip/defaults/main.yml

```diff
@@ -20,3 +20,18 @@ kube_vip_interface: "{{ keepalived_interface | default(kubernetes_keepalived_int
 
 # IP address to use for kube-vip
 kube_vip_address: "{{ keepalived_vip | default(kubernetes_keepalived_vip) }}"
+
+# use bgp for kube-vip
+kube_vip_bgp_enabled: false
```

Comment:
I think we should create something like `kube_vip_mode` which can either be `arp` or `bgp`.